MAUI/Maps/bubble.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 Bubbles can be rendered in different colors and sizes based on the data values of their assigned shape. You can add information to shapes such as population density, number of users, and more. 
 
+To learn more about our .NET MAUI Maps bubbles, you can check the below video.
```

Comment:
To learn more about the .NET MAUI Maps bubbles, you can check the following video.